.github/workflows/accept-promotion-to-master.yaml

```diff
@@ -0,0 +1,22 @@
+name: Accept promotion PRs to master
+
+on:
+  check_suite:
+    types: 
+      - completed
```

Comment:
The [`check_suite` event](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#check-suite-event-check_suite) isn't tied to a PR. We want this to trigger on PR events.

The events that the `automerge-action` authors recommend are:
```
    pull_request:
        types:
            - labeled
            - unlabeled
            - synchronize
            - opened
            - edited
            - ready_for_review
            - reopened
            - unlocked
    pull_request_review:
        types:
            - submitted
```

The `pull_request` events make sure that it triggers on possible modifiers of the pull request. I think that `pul_request_review.submitted` is the event that happens when a reviewer approves a PR.

Reply:
I'm working on this one.  If you watch the logs, these will trigger the even, but they will retry until all the checks pass.
https://github.com/pascalgn/automerge-action/pull/8 is floating around, but needs love.